src/app/js/formats/code/Component.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SyntaxHighlighter from 'react-syntax-highlighter';
+import { tomorrow } from 'react-syntax-highlighter/styles/hljs';
+import { field as fieldPropTypes } from '../../propTypes';
+
+const Component = ({ resource, field }) => {
+    const languageToHighlight =
+        field.format &&
```

Comment:
use lodash.get

```js
get(field, 'format.args.languageToHighlight', '');
```
will do the same

Reply:
Or better yet add a mapStateToProps and use fromField.getFieldFormatArgs, see BarChartView for an exemple